### DISCUSSION TITLE
Installing Snakemake for metaWrap Environment?

### DISCUSSION THREAD

**Discussion by tanio003** [franciscozorrilla/thesis_pipeline#101] [Q&A] [answered]
Hi Francisco,
How do I install/use Snakemake for the metaWrap environment, which requires Python ver2? I tried using Conda to install Snakemake, but it says it requires Python > 3.7. Even the older versions of Snakemake require at least Python 3.2.

In that case, should I not use Snakemake for metaWrap related tasks?

Tatsuro Tanioka

**Comment by franciscozorrilla** [comment, marked as answer]
Dear Tatsuro,

I apologize for the extremely delayed response, apparently I do not get notified when somone opens a new discussion 🤔
Hopefully this issue has not been troubling you, and I think you would have figured it out after playing around with metaGEM for a little bit.

In short, you do not need to have Snakemake installed in the metawrap environment, since that env is not used for launching jobs with Snakemake, rather it is only activated within the submitted metawrap jobs. For example, check out the tutorial to see how one can [submit metawrap jobs](https://github.com/franciscozorrilla/unseenbio_metaGEM#4-generate-final-bin-sets-using-metawrap-bin_refinement-and-bin_reassemble) from the metagem env.

I hope this clears things up and please let me know if you have follow up questions.

Best wishes,
Francisco

**Reply by tanio003** [reply]
Thank you! I got that figured out now.
